vip-api/src/main/java/fr/insalyon/creatis/vip/api/CarminProperties.java

```diff
@@ -64,7 +64,7 @@ public interface CarminProperties {
     // CARMIN PROCESSING
 
     String API_PIPELINE_WHITE_LIST = "carmin.processing.pipelines.whitelist";
-    String ADDITIONNAL_INPUT_VALID_CHARS = "\\[\\]?&="; // will be use in regex, so brackets must be escaped
+    String ADDITIONNAL_INPUT_VALID_CHARS = "\\[\\]"; // will be use in regex, so brackets must be escaped
```

Comment:
This has been removed, so the `?&=` are not necessary anymore ?

Reply:
Ah, I didn't see that it's deleted, I will fix this. It's necessary to send a URI in parameters.

Reply:
it's fixed, you can check now